bin/check-process.rb

```diff
@@ -287,7 +300,7 @@ def run
 
     if config[:metric]
       # #YELLOW
-      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, b| a + b } # rubocop:disable SingleLineBlockParams
+      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, e| a + e }
```

Comment:
I'll undo this change in a commit in just a second.

For your reference, the docs for rubocop for this rule are here:

http://rubocop.readthedocs.io/en/latest/cops_style/#stylesinglelineblockparams

The relevant section is `{"reduce"=>["acc", "elem"]}, {"inject"=>["acc", "elem"]}`.

I'm guessing `acc` there is the "acculumator value (_memo_) and `elem` is for "element" from the [Ruby Standard Library docs for `Enumerable`](https://ruby-doc.org/core-2.1.0/Enumerable.html#method-i-reduce). So shortening those names to `a` and `e` makes sense. Opinionated? Perhaps. But at least I can see where the logic comes from.

Either way, not trying to impose more rules than this repo's maintainers would care to adhere to, so I'll revert this change.